components/gitpod-cli/cmd/ssh.go

```diff
@@ -30,18 +29,14 @@ See %s/user/keys for a guide on setting them up.
 		ctx, cancel := context.WithTimeout(cmd.Context(), 10*time.Second)
 		defer cancel()
 
-		wsInfo, err := gitpod.GetWSInfo(ctx)
-		if err != nil {
+		if wsInfo, err := gitpod.GetWSInfo(ctx); err == nil {
```

Comment:
(nit) the code on left looks more like go https://danp.net/posts/reducing-go-nesting/

Reply:
Addressed in https://github.com/gitpod-io/gitpod/pull/18289/commits/5874d16f890933fbded5d818d658a77f794ad082 👍